frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -392,10 +392,28 @@ const StyledDrawer = styled(Drawer)`
   }
 `
 
+const MobileFacetSearchButtons = styled.div`
+  width: 100%;
+  gap: 12px;
+`
+
+const MobileApplyFiltersButton = styled(Button)`
+  background-color: ${({ theme }) => theme.custom.colors.mitRed};
+  color: ${({ theme }) => theme.custom.colors.white};
+  padding: 12px;
+  border-radius: 4px;
+  width: 144px;
+  margin-right: 12px;
+  box-shadow:
+    0 2px 4px 0 #25262b1a,
+    0 3px 8px 0 #25262b1f;
```

Comment:
The `<Button variant="primary" />` is missing its shadow. That's a bug. IMO, don't change it here. Just live with no shadow. I will create a separate issue for that.

Reply:
https://github.com/mitodl/hq/issues/4700